devicetypes/qubino/qubino-flush-2-relay.src/qubino-flush-2-relay.groovy

```diff
@@ -74,11 +74,11 @@ metadata {
 def installed() {
 	if (zwaveInfo?.model.equals("0051")) {
 		state.numberOfSwitches = 2
-	} else {
+	} else if (zwaveInfo?.model.equals("0052") && zwaveInfo?.model.equals("0053")){
```

Comment:
I guess you've meant to put `||` between those conditions. Also there should be space before curly bracket

Reply:
@PKacprowiczS thanks! Fixed